spec/20-http_request_header_format.md

```diff
@@ -297,7 +297,7 @@ The `tracestate` value is the concatenation of trace graph key/value pairs
 
 Example: `vendorname1=opaqueValue1,vendorname2=opaqueValue2`
 
-Only one entry per key is allowed because the entry represents that last position in the trace. Hence vendors must overwrite their entry upon reentry to their tracing system.
+Only one entry per key is allowed. Hence vendors MUST update their entry instead of adding it twice when propagating a modfied `tracestate` header downstream.
```

Comment:
* the word "hence" not needed
* instead of speaking of updating/adding, I would rather say vendors MUST ensure that the entry with their vendor key is only present once in the header sent downstream.

Reply:
+1 to just say that vendor is responsible for not duplicating keys it owns

Reply:
Well, that is exactly what I have added down in the "Mutating the traceparent Field" section -> https://github.com/w3c/trace-context/pull/477/files#diff-3832602716774745d030f22592f4c5b7d8f316e4d44be0165282244f9d6f14d9R358

Do we just want to duplicate that phrasing here? Seems redundant. How about doing away with the second sentence entirely? So just "Only one entry per key is allowed." and that's it. We also have the cross-reference 

> See [Mutating the tracestate Field](#mutating-the-tracestate-field) for details. 

so I think spelling out specifics here too is not required, the actual normative rules belong in the "Mutating" section and are already phrased as you requested.

WDYT?

Reply:
Yes, perhaps just delete the entire phrase including "Only one entry per key is allowed."

Reply:
I kept the first sentence, it leads nicely into the following example, but I removed the duplication about not adding your own header twice.